hack/setup-test-cluster.sh

```diff
@@ -5,6 +5,8 @@ set -e
 # expect oc to be in PATH by default
 export OC_TOOL="${OC_TOOL:-oc}"
 
+export NON_PTP_LABEL="node-role.kubernetes.io/virtual"
```

Comment:
do you want to use a default value perhaps like in OC_TOOL above ?  e.g. `"${NON_PTP_LABEL:-node-role.kubernetes.io/virtual}"`  not sure about the syntax with special chars, you'll have to check me on that.

Reply:
yes, of course, you are right, that was the whole point, silly me..

Reply:
tnx for catching this